datatypes/dataservice_pull_down_menu.md

```diff
@@ -123,20 +123,20 @@ are not included in Data service.
 - ergo:zipCodeArea
 - incore:ADT
 - incore:EPNRecoveryVer1
-- incore:JoplinCGE
-- incore:JoplinCGEbb
-- incore:JoplinCGEemploy
-- incore:JoplinCGEigtd
-- incore:JoplinCGEiout
-- incore:JoplinCGEjobcr
-- incore:JoplinCGElandcap
-- incore:JoplinCGEmisc
-- incore:JoplinCGEmisch
-- incore:JoplinCGEoutcr
-- incore:JoplinCGEsam
-- incore:JoplinCGEshocks
-- incore:JoplinCGEtauff
-- incore:JoplinCGEtpc 
+- incore:CGE
```

Comment:
Is there an incore:CGE type?

Reply:
no, it was always incore:CGE+something. so eventually, all.md or dataservice_pull_down_menu file, there is only one incoer:CGE

Reply:
Should that be removed then? This seems to imply there is an incore:CGE type.

Reply:
removed from both all and dataservice_pull_down_menu